File: Wordle/Word.java

public class Word {
	private int position;
	private char character;

	
	public Word(int pos, char chara) {
		position = pos;
		character = chara;

	}

	public int getLocation() { return position; }
	public void setLocation(int l) { position = l; }
    public char getCharacter() {return character;}
    public void setCharacter(char chara) {character = chara;}
}

File: Frontend/lib/pages/wordleGame/services/word.dart
class Word{
  int ?position;
  String ?character;
  
  Word({required this.position, required this.character});
  
  getLocation(){
    return position;
  }
  setLocation(){
    position = 1;
  }

  getCharacter(){
    return character;
  }
  setCharacter(String chara){
    character = chara;
  }

}
  

  